rules/windows/defense_evasion_microsoft_defender_tampering.toml

```diff
@@ -0,0 +1,72 @@
+[metadata]
+creation_date = "2021/10/18"
+maturity = "production"
+updated_date = "2021/12/13"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when one or more features on Microsoft Defender is disabled. Adversaries may maliciously modify Microsoft Defender registry 
+in order to hinder or disable Windows Defender mechanisms. 
+"""
+false_positives = ["Planned Windows Defender configuration changes can cause this detection to trigger."]
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Microsoft Windows Defender Protection Feature Tampering"
+note = """## Triage and analysis
+
+Detections should be investigated to identify if the hosts and users are authorized to use this tool. Endpoints should be isolated.  
```

Comment:
Which tool? Defender? Needs to be fixed

Reply:
@austinsonger, Please only mark conversation as resolved after taking action and doing the suggested, or after discussing it.

Reply:
Well I thought I did... So now I'm confused?